cpu/stm32f1/periph/i2c.c

```diff
@@ -141,8 +141,8 @@ static void _pin_config(gpio_t scl, gpio_t sda)
 {
     /* toggle pins to reset analog filter -> see datasheet */
     /* set as output */
-    gpio_init(scl, GPIO_DIR_OUT, GPIO_NOPULL);
-    gpio_init(sda, GPIO_DIR_OUT, GPIO_NOPULL);
+    gpio_init(scl, GPIO_OUT);
```

Comment:
Actually, `GPIO_OUT` is correct. the `OD` mode is set only after toggling the pins in linex 154-155...